app/src/main/java/com/github/godspeed010/weblib/LibraryFragment.kt

```diff
@@ -97,6 +90,23 @@ class LibraryFragment : Fragment() {
         return false
     }
 
+    private fun addColorsToDialog(radioGroup: RadioGroup)
+    {
+        enumValues<FolderColor>().forEach {
+            val button = RadioButton(context)
+            button.text = ""
+            button.tag = it
+            button.buttonTintList = ColorStateList(
+                arrayOf(intArrayOf(android.R.attr.state_enabled)),
+                intArrayOf(ContextCompat.getColor(requireContext(), it.rgbId))
+            )
+            button.layoutParams = LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+            (button.layoutParams as LinearLayout.LayoutParams).weight = 1f
+            radioGroup.addView(button)
```

Comment:
I noticed that when I rotated my phone into Landscape Mode, the RadioGroup was not centered and leaning towards the left of the screen. 

Just re-looked at the Portrait mode too, and that's also off-center, although not as noticeable.

Please make both of them centered.

Reply:
This involved a bit more than I was expecting, though I realized later what was causing it to have the additional spacing after it, for some reason having a weight different than 0 made a small space to the right of each radio button. I also ended up migrating the FolderColor radiogroup into a custom LinearLayout to simplify the interface.